instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/utils/history/MapboxHistoryTestRule.kt

```diff
@@ -70,14 +72,10 @@ class MapboxHistoryTestRule : TestWatcher() {
         try {
             runner()
         } catch (t: Throwable) {
-            val countDownLatch = CountDownLatch(1)
-            runOnMainSync {
-                historyRecorder.stopRecording {
-                    logE("$message history path=$it", "DEBUG")
-                    countDownLatch.countDown()
-                }
+            runBlocking {
```

Comment:
Shouldn't you specify `Dispatchers.Main` here?

Reply:
I think so, yes.